p4_symbolic/ir/expected/basic.pb.txt

```diff
@@ -0,0 +1,61 @@
+Could not transform to IR: UNIMPLEMENTED: Unsupported statement in action MyIngress.drop, found fields {
```

Comment:
This is up to you guys, but here are my two cents:

I really like not having many files that you need to look at when you review the golden files, so I personally would not want a .output and a .expected.output. I think here what I would do is: not name the file pb.txt, but simply <name>.expected. It can contain a protobuf if it worked, or an error message if it didn't.

In pdpi we go a step fruther, and the expected file actually contains both the input (that is just coming from the input of the test, so it's basically a copy), plus either output or error. That's nice, because now you can completely ignore the test input file, and just look at expected.  Here's an example:

```
=========================================================================
PacketIoTest: packet-in
=========================================================================

-------------------------------------------------------------------------
ok
-------------------------------------------------------------------------

payload: "1"
metadata {
  metadata_id: 1
  value: "4"
}
metadata {
  metadata_id: 2
  value: "eth-1/2/3"
}

payload: "1"
metadata {
  name: "ingress_port"
  value {
    hex_str: "0034"
  }
}
metadata {
  name: "target_egress_port"
  value {
    str: "eth-1/2/3"
  }
}
-------------------------------------------------------------------------
duplicate id
-------------------------------------------------------------------------

payload: "1"
metadata {
  metadata_id: 1
  value: "4"
}
metadata {
  metadata_id: 1
  value: "4"
}

Subtest failed with error:
  INVALID_ARGUMENT: Duplicate packet-in metadata found with ID 1. 
```

Not saying we have to go all the way to what pdpi does, but I personally am not a big fan of having to look in 50 different places to understand the test.

Reply:
One thing to keep in mind is that this is more or less temporary, after the next iteration, we should only have proto outputs.

The reason we are getting this here is simply because some features are unsupported for now.

Reply:
Perhaps it is best to remove the test cases that are not supported (what are they testing anyway?) and keep the one that is supported. As we add more features, we add more tests.

Reply:
I agree with @stefanheule that we would ideally want to see both input and output.
I also agree with @KinanBab, thanks for clarifying. Could we do what you suggest and also open an issue so we remember to address this another time? That way we don't block this PR.